lib/Launcher.js

```diff
@@ -72,7 +72,7 @@ class Launcher {
     if (typeof chromeExecutable !== 'string') {
       let chromiumRevision = require('../package.json').puppeteer.chromium_revision;
       let revisionInfo = Downloader.revisionInfo(Downloader.currentPlatform(), chromiumRevision);
-      console.assert(revisionInfo, 'Chromium revision is not downloaded. Run npm install');
+      console.assert(revisionInfo.downloaded, `Chromium revision is not downloaded. Run "npm install", or download manually from ${revisionInfo.url} and extract into ${revisionInfo.folderPath}.`);
```

Comment:
Let's leave this how it was. If npm install fails, it will tell them. And if chromium didn't download, then other dependencies are probably missing.